poi.config.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
         {
           resourceQuery: /blockType=readme/,
           loader: require.resolve('./example/vue-readme-loader')
-        }
+        },
+        { test: /\.flow$/, loader: 'ignore-loader' }
```

Comment:
why is this needed?

Reply:
I'm not really experienced frontend developer. But `date-fns v2` have `.js.flow` files in its locales and WebPack needs a loader for these files.
You can remove this line and try to run `yarn example`.

Reply:
webpack will only process `index.js`, `index.js.flow` isn't used.

Reply:
You can remove this line and try to run `yarn example`.
Webpack will throw an error for this file type.